server/models/user.ts

```diff
@@ -0,0 +1,92 @@
+"use strict";
+import { Model, UUIDV4, DataTypes } from "sequelize";
+
+
+interface UserAttributes {
```

Comment:
Not sure if these interfaces are useful unless we are going to be using them to enforce attributes for other Classes that we define. If we are just using a single interface for a single class, this may be redundant.

Reply:
The sequelize docs says to define your own models like this